test/c3/appframework/appframework_test.go

```diff
@@ -211,4 +212,102 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, false, true)
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("c3, appframework: can deploy a C3 SVA and have ES app installed on SHC", func() {
+
+			// ES is a huge file, we configure it here rather than in BeforeSuite/BeforeEach to save time for other tests
+			// Upload ES app to S3
+			esApp := "SplunkEnterpriseSecuritySuite"
+			appListV1 = append(testenv.BasicApps, esApp)
```

Comment:
Is this test covering basics apps as well ? Is that required ?

Reply:
`appListV1` is already initialized to testenv.BasicApps.  `appListV1 = append(appListV1, "SplunkEnterpriseSecuritySuite")` would achieve the same as 221-222. This applies if the answer to above query is `yes`